### ISSUE TITLE
( RAWR ) Error when removing the splash screen! Could not find color or style 'upmost'.

### ISSUE THREAD

**Issue by Jove125** [kiraio-moe/USSR#19]
Hi,

Error in the last version of USSR (in the previous version there was no):
```
( INFO ) Unity Version: 2021.2.19f1
( INFO ) Start removing Unity splash screen...
( INFO ) There's 2 splash screen detected.
( RAWR ) Error when removing the splash screen! Could not find color or style 'upmost'.
( INFO ) Cleaning up temporary files...
```

**Comment by kiraio-moe** [comment]
Can you attach the file?

**Comment by Jove125** [comment]
Same error on WebGL and Android versions
[WebGL.data.zip](https://github.com/kiraio-moe/USSR/files/14090387/WebGL.data.zip)

**Comment by pacolaf** [comment]
Same here with v1.1.7, in my case with iOS release and latest Unity LTS (2022.3.18f1)
( INFO ) Unity Version: 2022.3.18f1
( INFO ) Start removing Unity splash screen...
( RAWR ) Error when removing the splash screen! Unable to read beyond the end of the stream.
( INFO ) Cleaning up temporary files...

**Comment by kiraio-moe** [comment]
Hmm, many bug fixes and many bug's coming.. hah~

**Comment by shole** [comment]
The `Could not find color or style 'upmost'` issue occurs when drawmode is set to 'all sequential'.
Works as expected when set to 'Unity logo below'.
This was with 2020.3.48

**Comment by shole** [comment]
ah, apparently it occurs if you have any custom logos on either setting.

**Comment by kiraio-moe** [comment]
@Jove125, @pacolaf , @shole Fixed in 31ed4f51d90751ed97db48b310ea1c72bb8f3cb4.
